src/hooks/useStakingValidator.ts

```diff
@@ -55,31 +107,28 @@ export const useStakingValidator = () => {
         unbondingDelegations?.some((d) => d.validator === validator.operatorAddress.toLowerCase())
       ) ?? [];
 
-    if (!filteredValidators || filteredValidators.length === 0) {
-      return undefined;
-    }
-
-    // Find the validator with the fewest tokens
-    const validatorWithFewestTokens = filteredValidators.reduce((prev, curr) => {
-      return BigInt(curr.tokens) < BigInt(prev.tokens) ? curr : prev;
-    });
-
     return {
-      selectedValidator: validatorWithFewestTokens,
-      availableValidators: filteredValidators, // TODO: OTE-434
+      validatorWithFewestTokens,
```

Comment:
[q] do we still use this? 👀

Reply:
ah ur right - unnecessary - will remove!